server/app/controllers/dev/FeatureFlagOverrideController.java

```diff
@@ -1,40 +1,44 @@
 package controllers.dev;
 
-import annotations.FeatureFlagOverrides;
-import auth.Authorizers;
+import auth.Authorizers.Labels;
 import com.typesafe.config.Config;
+import featureflags.FeatureFlags;
 import javax.inject.Inject;
 import org.pac4j.play.java.Secure;
 import play.Environment;
+import play.mvc.Http.HeaderNames;
 import play.mvc.Http.Request;
 import play.mvc.Result;
 
-/** Allows for overriding of feature flags by the CiviForm Admin via HTTP request. */
+/**
+ * Allows for overriding of feature flags by an Admin via HTTP request.
+ *
+ * <p>Overrides are stored in the session cookie and used by {@link FeatureFlags} to control system
+ * behavior
+ */
 public final class FeatureFlagOverrideController extends DevController {
-  private final FeatureFlagOverrides overrides;
 
   @Inject
-  public FeatureFlagOverrideController(
-      FeatureFlagOverrides overrides, Environment environment, Config configuration) {
+  public FeatureFlagOverrideController(Environment environment, Config configuration) {
     super(environment, configuration);
-    this.overrides = overrides;
   }
 
-  @Secure(authorizers = Authorizers.Labels.CIVIFORM_ADMIN)
+  @Secure(authorizers = Labels.ANY_ADMIN)
```

Comment:
Definitely doesn't matter right now, but we might want to allow anyone to call this since we already protect it based on whether we're a dev environment or not. As currently configured, we can't force an experiment as an applicant.